UKHomeOffice/sgmr-service
# SPCR-501 Fix scroll to error not working on CYA page

### AC / Description of changes
Currently when there are errors on the check your answers page, the error links do not scroll to the error. Ensure that when the links in the error summary scroll to the correct section of the CYA page

### To Test
- Create a voyage
- Add some info to the departure form
- Save and exit
- Return to the draft
- On the CYA page click submit
- On the error summary click the links
- > The links should scroll to the correct section

### Notes

---

**Remember to merge to feature branches not `master`.**
